modules/rosa-policy-incident.adoc

```diff
@@ -55,7 +55,7 @@ All {product-title} clusters are backed up using AWS snapshots. Notably, this do
 .2+|Full object store backup, all SRE-managed cluster PVs
 |Daily
 |7 days
-.2+|This is a full backup of all Kubernetes objects, such as etcd, and all SRE-managed PVs in the cluster.
+.2+|This is a full backup of all Kubernetes objects, such as etcd, and all SRE-managed PVs in the cluster. Etcd backup does not apply to STS clusters.
```

Comment:
Not sure what the guide says but does it make sense to place that in parenthesis since it's more of a note?

Reply:
I'd be inclined to do so, but the ISG says to avoid parenthetical statements: "Use parentheses to identify items such as abbreviations, symbols, and measurements, but avoid using parentheses in general text."